src/main/java/org/opensearch/security/user/UserService.java

```diff
@@ -251,7 +252,7 @@ public String generateAuthToken(String accountName) throws IOException {
 
         String authToken = null;
         try {
-            DefaultObjectMapper mapper = new DefaultObjectMapper();
```

Comment:
it is not nice thing. `DefaultObjectMapper` should have a private constructor to avoid of such things